src/crud.py

```diff
@@ -329,17 +329,18 @@ def create_max_calls_per_sponsee_condition(
     db_condition = models.Condition(
         **{
             "type": schemas.ConditionType.MAX_CALLS_PER_SPONSEE,
-            "sponsee_address": condition.sponsee_address,
+            "contract_id": condition.contract_id,
```

Comment:
why you can't write a migration script ? 
If you have a database with the last version of model you can write a migration file and execute it I think.

Reply:
It's more of a "I won't bother for now" thing.